cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -2054,8 +2055,8 @@ impl_runtime_apis! {
 
 			parameter_types! {
 				pub const TrustedTeleporter: Option<(Location, Asset)> = Some((
-					WestendLocation::get(),
-					Asset { fun: Fungible(UNITS), id: AssetId(WestendLocation::get()) },
+					AssetHubWestend::get(),
+					Asset { fun: Fungible(UNITS), id: AssetId(AssetHubWestend::get()) },
```

Comment:
we want to simulate scenario, where `RandomParaLocation` can teleport us native token aka WND aka WestendLocation

```suggestion
					RandomParaLocation::get(),
					Asset { fun: Fungible(UNITS), id: AssetId(WestendLocation::get()) },
```